internal/kubernetes/operator/project/project.go

```diff
@@ -198,6 +200,18 @@ func BuildAtlasProject(br *AtlasProjectBuildRequest) (*AtlasProjectResult, error
 		result.Teams = teams
 	}
 
+	if br.Validator.FeatureExist(features.ResourceAtlasProject, featureBCP) {
```

Comment:
Nice you used the validator! I'll also add this in my PR. Thank you for the idea!